apps/client/src/components/BookingInfo/BookingInfo.tsx

```diff
@@ -74,6 +74,12 @@ export const BookingInfo: React.FC<BookingInfoProps> = ({
             data={bookingMomentDate.format('HH:mm - DD MMMM YYYY')}
             sx={{ alignItems: 'center', mb: 2 }}
           />
+          {vpoReferenceNumbers?.length && (
+            <Typography variant="h6" mb={4}>
+              {BOOKING.bookingInfoReferenceNumbers}
+              {vpoReferenceNumbers.join(', ')}
```

Comment:
Does this look like a long line of numbers? I mean, is it readable?

Reply:
What do you propose? Numbered list? The bottom may be cut on the phone's screenshots.

Reply:
done